docs: add MAINTAINING.md with maintenance guidelines

zetok opened the pull request:
Most likely doesn't have all the necessary knowledge (yet).

cc people with write access:
@agilob 
@antis81 
@dubslow 
@sudden6 
@TheSpiritXIII 
@tux3 

Should something be changed / clarified?

zetok committed:
docs: add MAINTAINING.md with maintenance guidelines

Most likely doesn't have all the necessary knowledge (yet).

sudden6 commented:
could you add the `.gitconfig` line to automatically sign every commit? It is very hard to find elsewhere.

SkyzohKey commented on MAINTAINING.md:
```diff
@@ -0,0 +1,80 @@
+
+# Commits
+
+- **always** use [commit message format]
```
Link to CONTRIBUTING.md is missing ;p

zetok replied:
Fixed.

sudden6 commented on MAINTAINING.md:
```diff
@@ -0,0 +1,81 @@
+
+Read it: https://help.github.com/articles/signing-commits-using-gpg/
+
+After you read it, perhaps including "Further reading", do what tip tells you.
```
I think there's a `the` missing in `do what tip tells you.`

zetok replied:
Fixed.

zetok committed:
docs(MAINTAINING.md): add more git-related info

Point to CONTRIBUTING.md section for GPG-signing config & links.
Add command for alias to check commit signatures.

sudden6 commented on MAINTAINING.md:
```diff
@@ -0,0 +1,94 @@
+
+# Git config
+
+## GPG signing
```
Maybe point to CONTRIBUTING.md where the setup is already explained?

zetok replied:
Yep, when it will be explaining, i.e. on master :)

sudden6 replied:
Well, now it is on master ;)

zetok replied:
Yep, done :)
